tools/serverpod_cli/test/database/extentions/defaults/bool_default_column_test.dart

```diff
@@ -92,7 +92,7 @@ void main() {
           () {
         expect(
           defaultColumn.toPgSqlFragment(),
-          '"boolDefault" boolean DEFAULT TRUE',
+          '"boolDefault" boolean DEFAULT true',
```

Comment:
Because this is how its how it is stored in posgress:
![Screenshot 2024-07-23 at 9 22 11 AM](https://github.com/user-attachments/assets/db69bd3f-4f40-4966-84c8-cfa2ea9ddd0d)